src/common/utils/include/os_macros.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+// TODO: Double check this Critical error handling
+/* for calloc: x = calloc(4,sizeof(char)) -> os_calloc(4,sizeof(char),x) */

```

Comment:
Should this be removed?

Reply:
That's a legacy comment. I dont know, and it is beyond the scope of this PR